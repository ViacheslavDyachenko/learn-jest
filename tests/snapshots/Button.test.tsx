import renderer from 'react-test-renderer';
import Button from './Button';
import React from 'react';

test('компонент Button корректно отображается на экране', () => {
    const button = renderer.create(<Button disabled={false} text='Кнопка' type='primary' btnClick={() => {}} />);
    let tree = button.toJSON();
    expect(tree).toMatchSnapshot();
})