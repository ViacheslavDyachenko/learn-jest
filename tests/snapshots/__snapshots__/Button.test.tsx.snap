// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`компонент Button корректно отображается на экране 1`] = `
<button
  className="primary"
  disabled={false}
  onClick={[Function]}
>
  Кнопка
</button>
`;
